android/src/main/java/com/urbanairship/reactnative/ReactNotificationProvider.java

```diff
@@ -0,0 +1,102 @@
+/* Copyright Urban Airship and Contributors */
+
+package com.urbanairship.reactnative;
+
+import android.content.Context;
+import android.graphics.BitmapFactory;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.support.annotation.WorkerThread;
+import android.support.v4.app.NotificationCompat;
+
+import com.urbanairship.AirshipConfigOptions;
+import com.urbanairship.Logger;
+import com.urbanairship.UAirship;
+import com.urbanairship.push.PushMessage;
+import com.urbanairship.push.notifications.AirshipNotificationProvider;
+import com.urbanairship.push.notifications.NotificationArguments;
+
+public class ReactNotificationProvider extends AirshipNotificationProvider {
```

Comment:
You should update to SDK 10.0.1 and override the getters

Reply:
Updated.